server/index.js

```diff
@@ -32,7 +35,7 @@ app.use('/:id', express.static(__dirname + '../../public'));
 // RUD controllers
 app.use('/description/:productId', hashParam);
 app.route('/description/:productId')
```

Comment:
I think I might rename all of my endpoints and refactor the express server to use route because this is so much cleaner than what I have right now.

Reply:
it's a little extra overhead but makes managing and scaling up endpoints much cleaner and more readable